### DISCUSSION TITLE
Read errors/alerts from Solis?

### DISCUSSION THREAD

**Discussion by J14111** [wills106/homeassistant-solax-modbus#258] [Q&A]
Is it possible to ready any errors/alerts from a Solis inverter?

**Comment by wills106** [comment]
It's on the list #200 

Most errors are stored as individual bits. You can do a crude representation of all the bits as decimal numbers and build a corresponding list.
I don't know if it's possible to get more than one error at a time. So ideally there needs to be proper bit decoding and a way to stack errors as a sensor if there is more than one error at a time.
I haven't really given it much attention yet as I would like to think Inverters don't go into an error state that often.

**Reply by J14111** [reply]
thanks wills, yeah hopefully it wont happen very often but it would be good to even have a heads up so I can go check the screen etc
